geohabnet/R/params.R

```diff
@@ -1,12 +1,12 @@
 
 #' Get Parameters
 #'
-#' Retrieves the parameters and copies the parameter file to the specified
+#' This function retrieves the parameters and copies the parameter file to the specified
 #' output path.
 #' @param out_path character. The output path where the parameter file will be
-#' copied. Default is temporary directory [tempdir()]
-#' @param iwindow logical. If `TRUE`, prompts the user to select the output
-#' directory using a file chooser window. Default is `FALSE`
+#' copied. The default is a temporary directory [tempdir()]
+#' @param iwindow logical. If window = `TRUE`, this will prompt the user to select the output
+#' directory using a file chooser window. The default parameter is set to `FALSE`.
```

Comment:
```suggestion
#' directory using a file chooser window. The default value is `FALSE`.
```
`_parameter_` is a reference, _default_ is a value.